bot/utils/snakes/hatching.py

```diff
@@ -0,0 +1,44 @@
+h1 = '''```
+        ----
+       ------
+     /--------\\
+     |--------|
+     |--------|
+      \------/
+        ----```'''
+
+h2 = '''```
+        ----
+       ------
+     /---\\-/--\\
+     |-----\\--|
+     |--------|
+      \------/
+        ----```'''
+
+h3 = '''```
+        ----
+       ------
+     /---\\-/--\\
+     |-----\\--|
+     |-----/--|
+      \----\\-/
+        ----```'''
+
+h4 = '''```
+        -----
+       -----  \\
+     /--|  /---\\
+     |--\\  -\\---|
+     |--\\--/--  /
+      \------- /
+        ------```'''
+
+stages = [h1, h2, h3, h4]
+snakes = {
```

Comment:
Compared to the immense amount of snakes we have in other parts of the cogs, could we not find any more for this part?

Reply:
This is perhaps something that can be PR'd in later. I do agree that it should have more, but I'm not gonna prioritize it now.

Reply:
Sounds good to me